tests/data/test_data_preprocessing_utils.py

```diff
@@ -18,12 +23,30 @@
 
 # Local
 from tuning.config import configs
+from tuning.data.data_config import DataLoaderConfig, DataSetConfig
 from tuning.data.data_preprocessing_utils import (
     combine_sequence,
     get_data_collator,
     validate_data_args,
 )
-from tuning.data.setup_dataprocessor import is_pretokenized_dataset, process_dataargs
+from tuning.data.data_processors import get_dataprocessor
+from tuning.data.setup_dataprocessor import (
+    _process_dataconfig_file,
+    is_pretokenized_dataset,
+    process_dataargs,
+)
+
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
+PREDEFINED_DATA_CONFIGS = os.path.join(BASE_DIR, "examples", "predefined_data_configs")
+APPLY_CUSTOM_TEMPLATE_YAML = os.path.join(
+    PREDEFINED_DATA_CONFIGS, "apply_custom_template.yaml"
+)
+PRETOKENIZE_JSON_DATA_YAML = os.path.join(
+    PREDEFINED_DATA_CONFIGS, "pretokenized_json_data.yaml"
+)
+TOKENIZE_AND_INSTRUCTION_MASKING_YAML = os.path.join(
+    PREDEFINED_DATA_CONFIGS, "tokenize_and_instruction_masking.yaml"
+)
```

Comment:
Should this go in an \_\_init\_\_.py file like the one for [tests/testdata](https://github.com/dushyantbehl/fms-hf-tuning/blob/c078d071beeadef718dd4d645655658ae5391e9d/tests/testdata/__init__.py)?

Reply:
Yea make sense. Thanks Will. Pushed the refactored code.